axelrod/data/ann_weights.csv

```diff
@@ -0,0 +1,5 @@
+# name, features, hidden_layer_size, weights...
```

Comment:
Small suggestion: remove the `#` and list full header (potentially useful for other analysis?). I expect this would need to be done on the `axelrod-evolver` repo and not for this PR.